lib/db2.js

```diff
@@ -88,6 +90,30 @@ DB2.prototype.update = function(model, where, data, options, cb) {
 };
 
 /**
+ * Update all instances that match the where clause with the given data
```

Comment:
While technically not wrong I will change it to "Update the instance matching the ID in the where clause".  Note that if the id field is not unique there could be more than one and loopback does not take that into account.